activation/activation_test.go

```diff
@@ -320,6 +320,7 @@ func publishAtx(b *Builder, meshLayer types.LayerID, clockEpoch types.EpochID, b
 // ========== Tests ==========
 
 func TestBuilder_PublishActivationTx_HappyFlow(t *testing.T) {
+	types.SetLayersPerEpoch(int32(layersPerEpoch))
```

Comment:
you have the init line so you don't need this

Reply:
I do since other tests change it unfortunatly